backend-api/src/functions/jwks/jwksBuilder/jwksBuilder.ts

```diff
@@ -0,0 +1,139 @@
+import {
+  GetPublicKeyCommand,
+  GetPublicKeyCommandOutput,
+  KMSClient,
+} from "@aws-sdk/client-kms";
+import { NodeHttpHandler } from "@aws-sdk/node-http-handler";
```

Comment:
```suggestion
import { NodeHttpHandler } from "@smithy/node-http-handler";
```

Reply:
Believe the recommendation for version 3 of the `aws-sdk` is to use smithy for the `node-http-handler`

Reply:
Updated